Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -602,237 +751,641 @@
+          "comparison": "isEqualTo",
+          "value": "0"
+        },
+        {
+          "parameterName": "param_check_provider_error",
+          "comparison": "isNotEqualTo",
+          "value": "-1"
+        }
+      ],
       "name": "Overview"
     },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "1edcbbbd-b9a6-4ad5-9a56-3f34b3cbd58c",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_check_error_availability",
+            "type": 1,
+            "query": "// 0 = no error\n// 1 = table does not exist (need to add provider)\n// 2 = table exists, but column v1 does not (need to wait)\nlet t = \"SapNetweaver_GetProcessList_CL\";\nlet c1 = \"hostname_s\";\nlet dummy = \"unmÃ¶glicher-wert!\";\nlet MissingTable = view () { print miss = 1, columnV1 = dummy, records = -1 };\nlet checkRecords =\n  union\n    isfuzzy=true MissingTable,\n    (table(t)\n      | summarize records = count()\n    )\n  | project records\n  | top 1 by records desc\n;\nlet checkTableExists =\n  union\n    isfuzzy=true MissingTable,\n    (table(t)\n      | count\n      | extend miss = 0\n    )\n  | project miss\n  | limit 2\n  | top 1 by miss asc\n;\nlet checkColumnExists =\n  union\n    isfuzzy=true MissingTable,\n    (table(t)\n      | extend miss = 0\n      | extend columnV1 = column_ifexists(c1, dummy)\n    )\n  | project miss, columnV1\n  | limit 2\n | top 1 by miss asc\n;\nlet tableExists = toscalar(checkTableExists | summarize max(miss) != \"1\");\nlet columnV1Exists = toscalar(checkColumnExists | summarize max(columnV1) != dummy);\nlet tableRecords = toscalar(checkRecords | summarize max(records));\nlet x = iif(tableExists, iif(columnV1Exists, iif(tableRecords > 0, \"0\", \"2\"), \"2\"), \"1\");\nprint toscalar(x)\n",
+            "isHiddenWhenLocked": true,
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "conditionalVisibilities": [
+        {
+          "parameterName": "selectedMenu",
+          "comparison": "isEqualTo",
+          "value": "Availability"
+        },
+        {
+          "parameterName": "dummy_neverset",
+          "comparison": "isNotEqualTo"
+        }
+      ],
+      "name": "ErrorCheckAvailParam"
+    },
     {
       "type": 12,
       "content": {
         "version": "NotebookGroup/1.0",
         "groupType": "editable",
         "items": [
           {
-            "type": 9,
+            "type": 1,
             "content": {
-              "version": "KqlParameterItem/1.0",
-              "parameters": [
+              "json": "Unable to create the required table (GetProcessList) to capture telemetry information. Please try after some time. If the issue persists, please contact support.",
```

Comment:
wording here seems off to me.  These workbooks themselves are never actually creating any tables right?  It seems like what we really want to say is that the expected table has not been created yet, which could indicate that data has never been uploaded.  Actually, what is the difference between this message and the message below?  In both cases the table does not exist, right?

Reply:
per discussion, harsha was just following Hana provider convention here.